ci/gpu/build.sh

```diff
@@ -49,6 +49,7 @@ conda install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvidia \
       "libcumlprims=0.15.0a200607" \
       "dask-cudf=${MINOR_VERSION}" \
       "dask-cuda=${MINOR_VERSION}" \
+      "dask-ml" \
```

Comment:
This should be added the `rapids-build-env`. Follow the steps here: https://docs.rapids.ai/maintainers/depmgmt/

Reply:
@raydouglass this is only needed to run the pytests. Should I still add it to the build recpes?

Reply:
@cjnolet Now that it's added to the `rapids-build-env`, you can remove it from here.